test/integration/manifests/cilium/cilium-nightly-config.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+data:
+  agent-not-ready-taint-key: node.cilium.io/agent-not-ready
+  arping-refresh-period: 30s
+  auto-direct-node-routes: "false"
+  bpf-lb-external-clusterip: "false"
+  bpf-lb-map-max: "65536"
+  bpf-lb-mode: snat
+  bpf-map-dynamic-size-ratio: "0.0025"
+  bpf-policy-map-max: "16384"
+  bpf-root: /sys/fs/bpf
+  cgroup-root: /run/cilium/cgroupv2
+  cilium-endpoint-gc-interval: 5m0s
```

Comment:
@camrynl for visibility , this needs to be changed for nightly pipeline. 
She already has a PR open for the Networking Aquarius. https://msazure.visualstudio.com/One/_git/Networking-Aquarius/pullrequest/8099539?_a=files
But we are moving to ACN so just want to make sure that we have the check here as well

Reply:
sounds good, I will be merging that PR soon. @jpayne3506 if you want to integrate that into this pr you can, or I can wait for this PR to merge and I can add this check in later

Reply:
I will let this PR merge without these listed changes. Thanks for communicating @camrynl @vipul-21 .